autodqm/dqm.py

```diff
@@ -23,6 +23,19 @@
 StreamProg = namedtuple('StreamProg', ('cur', 'total', 'path'))
 DQMRow = namedtuple('DQMRow', ('name', 'full_name', 'url', 'size', 'date'))
 
+## Map from subsystem to Online DQM directory
```

Comment:
Why is this OnlineMap required? As far as I understand, it reads this map to find the proper directory to grab the ROOT file. Perhaps we add a mapping file to the config directory instead of putting it in the dqm.py. This will be more obvious for future PRs

Reply:
I think a simple mapping in the configs is best. This way PRs in the future adding subsystems do not need to modify any python code.

Reply:
Hi @chadfreer, I've added the mapping here:
https://github.com/AutoDQM/AutoDQM/pull/70/commits/d5d177d3a2addc8564abeeb9b51ce25652807d96

Let me know if you see any issues.